### PR TITLE
Fix doc blocks / namespacing - everywhere

### PR DESCRIPTION
**Pull request by tomneedham**
(+ one bonus that was never actually included)

### CODE REVIEW

**Comment by butonic** [comment]
Fix all the doc blocks

File: apps/dav/lib/Connector/Sabre/Directory.php [hunk 1]
```diff
@@ -376,6 +387,12 @@ public function getQuotaInfo() {
      * @param string $fullSourcePath Full path to source node
      * @param INode $sourceNode Source node itself
      * @return bool
+	 * @throws BadRequest
+	 * @throws ServiceUnavailable
+	 * @throws \Sabre\DAV\Exception\Forbidden
```
Comment (DeepDiver1975) [inline apps/dav/lib/Connector/Sabre/Directory.php]: dup

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

File: apps/dav/lib/Connector/Sabre/Directory.php (hunk 1 above)
Comment (butonic) [reply apps/dav/lib/Connector/Sabre/Directory.php]: Not a duplicate: OCA\DAV\Connector\Sabre\Exception\Forbidden != \Sabre\DAV\Exception\Forbidden

**Review by butonic — APPROVED** [review]

File: apps/dav/lib/Connector/Sabre/Directory.php (hunk 1 above)
Comment (DeepDiver1975) [reply apps/dav/lib/Connector/Sabre/Directory.php]: then let's add a proper use statement

```php
use \Sabre\DAV\Exception\Forbidden as SabreForbidden

**Comment by tomneedham** [comment]
Fixed and prefixed sabre exceptions with 'Sabre'

**Comment by tomneedham** [comment]
horracle timeout

**Comment by tomneedham** [comment]
rebased for luck

**Review by DeepDiver1975 — APPROVED** [review]

**Comment by tomneedham** [comment]
rebased - waiting for CI

**Comment by tomneedham** [comment]
jenkins timeout - rebasing

**Pull request merged**